metricbeat/module/kvm/status/status.go

```diff
@@ -0,0 +1,172 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package status
+
+import (
+	"net"
+	"net/url"
+	"time"
+
+	"github.com/elastic/beats/v7/libbeat/common/cfgwarn"
+
+	"github.com/pkg/errors"
+
+	"github.com/digitalocean/go-libvirt"
+	"github.com/digitalocean/go-libvirt/libvirttest"
+
+	"github.com/elastic/beats/v7/libbeat/common"
+	"github.com/elastic/beats/v7/metricbeat/mb"
+)
+
+const (
+	// maximum number of memory stats to be collected
+	// limit is defined by REMOTE_DOMAIN_MEMORY_STATS_MAX
+	// based on https://github.com/libvirt/libvirt/blob/5bb07527c11a6123e044a5dfc48bdeccee144994/src/remote/remote_protocol.x#L136
+	maximumStats = 11
+	// flag VIR_DOMAIN_AFFECT_CURRENT passed to collect memory stats
+	// based on https://libvirt.org/html/libvirt-libvirt-domain.html#virDomainModificationImpact
+	flags = 0
+)
```

Comment:
@tomdoherty sorry, I was taking another look before merging and I have seen that it seems that these variables are not being used in this metricset. If they are not needed could you please remove them?

Reply:
fixed :-)